iot_verifier/src/loader.rs

```diff
@@ -145,7 +145,7 @@ impl Loader {
             .unwrap_or(window_default_lookback)
             .max(window_max_lookback);
         let before_max = after + self.window_width;
-        let before = (now - (self.window_width * 3)).min(before_max);
+        let before = (now - (self.window_width * 3)).max(before_max);
```

Comment:
i have a vague recollection of setting this to max deliberately but I have went through the logic again and cant see any reason why it should be.  

So the change is probably good but I will look at this flow further

Reply:
When set to max it would have made for a very wide load window in the scenario of backloading the reports after being down for a period of time.  That would be suboptimal and resulted in a potential memory issue as it would result in all beacons and witnesses being loaded for the period.

As such this change is good